nix/haskell.nix

```diff
@@ -204,20 +204,6 @@ haskell-nix: haskell-nix.cabalProject' [
               packages.servant-openapi3.components.setup.doExactConfig = true;
               packages.system-filepath.components.setup.doExactConfig = true;
 
-              packages.cardano-wallet-core.components.tests = {
-                unit.preCheck = noCacheTestFailuresCookie;
-                # Attempt to ensure visible progress in the macOS hydra job.
-                #
-                # An hypothesis is that #2472 is caused by heavy load and unfocused
-                # resources from running the tests concurrently, risking that the slowest
-                # hspec runner - and therefore the stdout - being silent for 900s causing
-                # hydra to timeout.
-                #
-                # Setting -j 1 should hopefully focus the resource we have in one place. It
-                # should go silent less often, at the expense of the full run getting slower.
-                unit.testFlags = lib.optionals pkgs.stdenv.hostPlatform.isDarwin [ "-j" "1" ];
```

Comment:
Right, we don't require Darwin to succeed anymore in bors, so not moving this setting should be fine 👍